.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**VPN version:**
```

Comment:
I see a lot of dup fields here. In theory, linking the jira issue should contain most of these fields.
The jira issue should be enough.

Reply:
This will be used if a User is filing a new issue. - The template is just copied from QA :D